core/creator/src/main/java/io/quarkus/creator/phase/augment/AugmentPhase.java

```diff
@@ -131,6 +132,18 @@ public AugmentPhase setWiringClassesDir(Path wiringClassesDir) {
         return this;
     }
 
+    /**
+     * The directory for generated classes. If none is set by the user,
+     * wiring-classes directory will be created in the work directory of the creator.
```

Comment:
This needs to be re-written for configDir.

Reply:
thanks, fixed it and added a check if configDir is null just to be sure.